ipatests/test_integration/test_commands.py

```diff
@@ -1134,3 +1134,34 @@ def test_cacert_manage(self):
             raiseonerr=False
         )
         assert result.returncode == 0
+
+    def test_ipa_adtrust_install_with_locale_issue8066(self):
+        """
+        This test checks that ipa-adtrust-install command runs successfully
+        on a system with locale en_IN.UTF-8 without displaying error below
+        'IndexError: list index out of range'
+
+        This is a testcase for Pagure issue
+        https://pagure.io/freeipa/issue/8066
+
+        """
+        # Set locale to en_IN.UTF-8
+        tasks.kinit_admin(self.master)
+        self.master.run_command(
+            ["localectl", "set-locale", "LC_TIME=en_IN.UTF-8"]
+        )
+        result = self.master.run_command(["localectl", "status"])
+        assert "LC_TIME=en_IN.UTF-8" in result.stdout_text
+        # Install ipa-server-adtrust and check status
+        msg1 = (
+            "Unexpected error - see /var/log/ipaserver-install.log"
+            "for details"
+        )
+        msg2 = "IndexError: list index out of range"
+        tasks.install_packages(self.master, ["*ipa-server-trust-ad"])
+        assert result.returncode == 0
```

Comment:
Is it for code at line 1153?. Can be removed.

Reply:
removed result.returncode == 0